core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+**[D1]**	 The contract SHOULD be an MSA between the contract parties. 
+
+An MSA is preferable since it allows a proliferation of contract based BPI (commercial) workflows and work staps between the parties reducing complexity and potential errors.
+
+**[CR1]< [D1]** 	There MUST be only one MSA between a contract parties covering commercial transactions for a given set of products, services or assets to disambiguate which terms cover which part of a commercial relationship between parties.
+
+Specific Terms and Conditions (“Specific T&C”) defines the terms and conditions governing a specific product, service or asset or set thereof offered and delivered by Provider(s) to Requester(s).
+
+**[D2]**	Each commercial state object SHOULD have its own Specific T&C document.
+
+This would allow the fine graining and consistent application of commercial state object specific business rules and data.
+
+### 2.4.2	Commercial Documents 
+
+Commercial Documents, a category of commercial state objects, refer to a specific product/service/asset or set thereof, which may or may not be modified from an original offering to meet the Requester requirements and includes operational and commercial details. A commercial document is an abstract construct representing mutual commitments.
+
+**[R4]**	A commercial state object to be transacted on MUST be based on a specific commercial document.
+
+**[R5]**	A commercial document MUST be represented as an electronic record. 
+
+**[R6]**	A commercial document MUST be authorized by legal representatives of the parties, or their legal delegates.
+
+**[D3]**	The definition of a commercial document authorization SHOULD be stated in the legal agreement underlying the commercial document.
+
+Authorizations for commercial transactions are a foundational element in the conext of this document, as they are in paper based agreements. Therefore, any legal authorization agreements relevant to the commercial agreement between commercial counterparties, and thus to commercial transactions between them, are important to be mapped onto a BPI to ensure mitigating the risk of unauthorized signatures.
+
+**[R7]**	The representatives and their authorized delegates who can perform commercial document authorizations SHOULD be explicitly listed or inferred from the stated legal delegation rules of the counterparties in the contract underlying any commercial document. 
+
+**[R8]**	A commercial document MUST be non-repudiable.
+
+Note that while non-repudiation in the physical world is most often tied to a physical signature of an individual on a legal document, in the digital world a digital signature over a digital legal document such as an Order or an Invoice belonging to a known and verifiable digital identity of a counter party serves the same purpose.
```

Comment:
@bitwiseguy a purchase order is a subset of orders, and has a very specific supply chain/P2P type meaning. This is meant to be more general. For example, I order a trade of an asset, and that is not a purchase order, but it could still baselines because it is a commercial order.